indico/htdocs/js/indico/modules/registration/reglists.js

```diff
@@ -247,7 +261,22 @@
         });
 
         $('.reglist-filter input:checkbox').on('change', function() {
-            colorizeFilter($(this).closest('.filter'));
+            colorizeFilter($(this).closest('.dropdown').siblings('.filter'));
+        });
+
+        $('.reglist-filter .title').on('mouseover', function() {
+            var title = $(this);
+            if (this.offsetWidth < this.scrollWidth) {
+                var qtip = title.qtip({hide: 'mouseout', content: title.text(), overwrite: false}).qtip('show');
```

Comment:
Why do you need to instantiate qtip manually? If it's just to have some text showing up on mouse-over you can simply set it as the `title` HTML attribute and it will appear automagically.